lib/elixir/lib/code/formatter.ex

```diff
@@ -1412,14 +1432,30 @@ defmodule Code.Formatter do
 
   defp bitstring_spec_to_algebra({op, _, [left, right]}, state) when op in [:-, :*] do
     {left, state} = bitstring_spec_to_algebra(left, state)
-    {right, state} = quoted_to_algebra_with_parens_if_operator(right, :parens_arg, state)
+    {right, state} = bitstring_spec_element_to_algebra(right, state)
     {concat(concat(left, Atom.to_string(op)), right), state}
   end
 
   defp bitstring_spec_to_algebra(spec, state) do
-    quoted_to_algebra_with_parens_if_operator(spec, :parens_arg, state)
+    bitstring_spec_element_to_algebra(spec, state)
+  end
+
+  defp bitstring_spec_element_to_algebra(
+         {atom, meta, empty_args},
+         state = %{normalize_bitstring_modifiers: true}
+       )
+       when is_atom(atom) and empty_args in [nil, []] do
+    empty_args = bitstring_spec_notmalized_empty_args(atom)
```

Comment:
Typo in this function name 🙃 

```suggestion
    empty_args = bitstring_spec_normalized_empty_args(atom)
```

Needs to be updated below too.

Reply:
Woops,  thank you 😅 Fixed